lib/pure/os.nim

```diff
@@ -1252,9 +1252,12 @@ elif defined(windows):
     if i < ownArgv.len and i >= 0: return TaintedString(ownArgv[i])
     raise newException(IndexError, "invalid index")
 
+elif defined(genode):
```

Comment:
That seems a bad implementation for paramStr and paramCount. Why not fail at compiletime instead?

Reply:
Those two had been undefined, I don't remember where the problem I was having was, but I believe libraries were calling those in an unavoidable way and it was easier to put conditionals in the application code.